__docs__/wonder-blocks-typography/body-monospace.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
```

Comment:
Creating stories for each typography component so that they are more discoverable. These won't be included in Chromatic tests. All the new stories for typography components are based on the same changes!